frontstage/templates/surveys/surveys-add.html

```diff
@@ -57,21 +57,21 @@ <h1 class="u-fs-xl ons-u-mt-s">Add a survey</h1>
                     "error": errorMsgText
                 })
             }}
-            <div class="ons-grid ons-grid--flex ons-grid--gutterless ons-grid--vertical-center ons-grid--no-wrap@s ons-u-mt-m ons-u-mb-m">
-                    <div class="ons-grid__col">
-                        {{
-                            onsButton({
-                                "text": "Continue",
-                                "type": "submit",
-                                "id": "continue_button",
-                                "submitType": "timer"
-                            })
-                        }}
-                    </div>
-                    <div class="ons-grid__col ons-u-ml-m">
-                        <a href="{{ url_for('surveys_bp.get_survey_list', tag='todo') }}" id="cancel_button" role="button">Cancel</a>
-                    </div>
+            <div class="ons-grid ons-grid-flex ons-grid--gutterless ons-grid-flex--vertical-center ons-grid--no-wrap@s ons-u-mt-m ons-u-mb-m">
+                <div class="ons-grid__col">
+                    {{
+                        onsButton({
```

Comment:
I can't see the change here and why onsButton is green on github

Reply:
If I'm correct the 'ons-grid-flex' has changed. Not sure why it has updated the onsButton as well.

Reply:
Yeah that's the change here, might be because both the buttons are in the same div that gets the `ons-grid-flex` update